r3bbase/pars/R3BMSOffsetPar.cxx

```diff
@@ -0,0 +1,77 @@
+/******************************************************************************
+ *   Copyright (C) 2019 GSI Helmholtzzentrum für Schwerionenforschung GmbH    *
+ *   Copyright (C) 2019-2023 Members of R3B Collaboration                     *
+ *                                                                            *
+ *             This software is distributed under the terms of the            *
+ *                 GNU General Public Licence (GPL) version 3,                *
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+
+#include "R3BMSOffsetPar.h"
+#include "R3BLogger.h"
+
+#include "FairDetParIo.h"
+#include "FairLogger.h"
+#include "FairParamList.h"
+
+#include "TArrayD.h"
+#include "TMath.h"
+#include "TString.h"
+
+// ---- Standard Constructor ---------------------------------------------------
+R3BMSOffsetPar::R3BMSOffsetPar(const char* name, const char* title, const char* context)
+    : FairParGenericSet(name, title, context)

```

Comment:
Try to use `std::string_view` rather than char pointer
```suggestion
R3BMSOffsetPar::R3BMSOffsetPar(std::string_view name, std::string_view title, std::string_view context)
    : FairParGenericSet(name.data(), title.data(), context.data())

```